hazelcast/src/main/java/com/hazelcast/jmx/ListMBean.java

```diff
@@ -29,27 +30,34 @@
 @ManagedDescription("IList")
 public class ListMBean extends HazelcastMBean<IList<?>> {
 
+    private final boolean totalEnabled;
+
     private AtomicLong totalAddedItemCount = new AtomicLong();
     private AtomicLong totalRemovedItemCount = new AtomicLong();
     private final String registrationId;
 
     protected ListMBean(IList<?> managedObject, ManagementService service) {
         super(managedObject, service);
+        totalEnabled = service.instance.node.groupProperties.ENABLE_JMX_DETAILED.getBoolean();
```

Comment:
I think the field `totalEnabled` and this line should be moved to `HazelcastMBean` class. This way we can get rid of code duplication in all other `..MBean` impls.